src/msg/msg_types.h

```diff
@@ -17,6 +17,10 @@
 
 #include <sstream>
 
+// #include <string> 
```

Comment:
please remove these `#include` if they are not related to this change.

Reply:
@tchaikov 
Done that already in local commits, they will be rebase in next commits